DancingLineECS/Assets/Scripts/Startup/MainSceneStartup.cs

```diff
@@ -0,0 +1,37 @@
+using ECS;
+using Leopotam.Ecs;
+using UnityEngine;
+
+namespace Startup
+{
+    public class MainSceneStartup : MonoBehaviour
+    {
+        private EcsSystems _systems;
+        private EcsWorld _world;
+    
+        private void Start()
+        {
+            _world = new EcsWorld();
+            _systems = new EcsSystems(_world);
+
+            var subsystems = GetComponentsInChildren<ISystemStartup>();
```

Comment:
nice decision, but keep in mind that having the whole systems order right in front of you has some benefits